bridge/cypress/support/pageobjects/DashboardPage.ts

```diff
@@ -19,7 +20,7 @@ class DashboardPage {
   }
 
   public clickProjectTile(projectName: string): EnvironmentPage {
-    cy.wait(500).get(this.PROJECT_TILE_LOCATOR.replace('proj_pattern', projectName)).click();
+    cy.wait('@projects').get(this.PROJECT_TILE_LOCATOR.replace('proj_pattern', projectName)).click();
```

Comment:
Thanks for removing the random wait time with an actual response to wait for :+1: